### PR TITLE
Fix Dependabot 138

### PR DESCRIPTION
**Pull request by zachstence**
### Description

Resolves https://github.com/evidence-dev/evidence/security/dependabot/138

### Checklist

- [ ] I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)

~~- [ ] For UI or styling changes, I have added a screenshot or gif showing before & after~~
~~- [ ] I have added to the docs where applicable~~
~~- [ ] I have added to the [VS Code extension](https://github.com/evidence-dev/evidence-vscode) where applicable~~

### CODE REVIEW

File: .github/workflows/lint-format.yml [hunk 1]
```diff
@@ -12,15 +12,15 @@ jobs:
...
       - name: Set up pnpm
         uses: pnpm/action-setup@v4.0.0
         with:
           version: 8.6.9
-      - name: Set up node v16
-        uses: actions/setup-node@v3
+      - name: Set up node
+        uses: actions/setup-node@v4
         with:
           node-version: 16
```
Comment (ItsMeBrianD) [inline .github/workflows/lint-format.yml:25]: Should this be updated?

File: .github/workflows/playwright-tests.yml [hunk 2]
```diff
@@ -30,13 +30,13 @@ jobs:
...
+        uses: actions/checkout@v4
         with:
           ref: ${{ github.event.pull_request.head.sha }}
       - uses: pnpm/action-setup@v4.0.0
         with:
           version: 8.6.9
-      - uses: actions/setup-node@v3
+      - uses: actions/setup-node@v4
         with:
           node-version: 16.x
```
Comment (ItsMeBrianD) [inline .github/workflows/playwright-tests.yml:41]: Same

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -24,7 +24,7 @@ jobs:
         with:
           version: 8.6.9
       - name: Setup Node.js
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
           node-version: 16.x
```
Comment (ItsMeBrianD) [inline .github/workflows/release.yml:29]: Also here

**Review by ItsMeBrianD — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 3 above)
Comment (zachstence) [reply .github/workflows/release.yml]: [d594a2d](https://github.com/evidence-dev/evidence/pull/2235/commits/d594a2d896613ae9afe1cf653648ec7dedbe4dfc)

File: .github/workflows/playwright-tests.yml (hunk 2 above)
Comment (zachstence) [reply .github/workflows/playwright-tests.yml]: [d594a2d](https://github.com/evidence-dev/evidence/pull/2235/commits/d594a2d896613ae9afe1cf653648ec7dedbe4dfc)

File: .github/workflows/lint-format.yml (hunk 1 above)
Comment (zachstence) [reply .github/workflows/lint-format.yml]: [d594a2d](https://github.com/evidence-dev/evidence/pull/2235/commits/d594a2d896613ae9afe1cf653648ec7dedbe4dfc)

**Review by ItsMeBrianD — APPROVED** [review]

**Pull request merged**